packages/web-client/src/helpers/share/permission.ts

```diff
@@ -41,11 +41,7 @@ export abstract class SharePermissions {
   )
 
   static readonly read = new SharePermission('read', SharePermissionBit.Read, $gettext('Read'))
-  static readonly update = new SharePermission(
-    'update',
-    SharePermissionBit.Update,
-    $gettext('Update')
-  )
+  static readonly update = new SharePermission('edit', SharePermissionBit.Update, $gettext('Edit'))
```

Comment:
```suggestion
  static readonly update = new SharePermission('update', SharePermissionBit.Update, $gettext('Edit'))
```

Reply:
Don't know if the key is used anywhere else, but we definitely only want to change the user facing label (the one that's wrapped in the `$gettext` helper)

Reply:
alright